src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1066,7 +1067,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                             ->validate()->castToArray()->end()
                         ->end()
                         ->scalarNode('translation_domain')->defaultValue('validators')->end()
-                        ->enumNode('email_validation_mode')->values(['html5', 'loose', 'strict'])->end()
+                        ->enumNode('email_validation_mode')->values(Email::VALIDATION_MODES)->end()
```

Comment:
`loose` is no longer allowed in the config, but this value is already rejected by the constraint class.

https://github.com/symfony/symfony/blob/ed0a2358835f87ae188cebee042009fedea2cc32/src/Symfony/Component/Validator/Constraints/Email.php#L28-L30

But this value is still used by default: https://github.com/symfony/symfony/blob/6b4f603edbd2bee7d340bdb6e57cf2b9c656096b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1660-L1662

Is there a bug here that makes this config required?

Reply:
This depends on the Symfony version you are browsing. The `loose` mode was deprecated in 6.2 that's why you don't see it anymore in the `7.3` branch. And yes, I think being able to still configure it through the semantic configuration also is a bug.

Reply:
but we should probably indeed make sure that applications still using `loose` don't break with the next patch release (as long as you don't use the `Email` constraint that's probably not a real issue today)